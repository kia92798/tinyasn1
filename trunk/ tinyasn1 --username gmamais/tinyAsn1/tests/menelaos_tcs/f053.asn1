F053 DEFINITIONS AUTOMATIC TAGS	
EXTENSIBILITY IMPLIED ::= BEGIN
ShortINTEGER ::= INTEGER (0..6)

Citizen ::= SET { country ENUMERATED {france, greece, italy, germany},
	    yearOfBirth	INTEGER,
	    married BOOLEAN,
	    nOfChildren ShortINTEGER}

	    GreekCitizen ::= Citizen (WITH COMPONENTS {...,
					country (greece)})    
	    aGreekCitizen GreekCitizen ::= {country greece, yearOfBirth 1980, married TRUE, nOfChildren 4}

	    GreekCitizenTwoChildren ::= GreekCitizen (WITH COMPONENTS {...,
					nOfChildren (2)})
	    aGreekCitizenWithTwoChildren GreekCitizenTwoChildren ::= {country greece, yearOfBirth 1980, married TRUE, nOfChildren 2}

		--this should cause a warning
	    aGreekCitizenWithTwoChildren3 GreekCitizenTwoChildren ::= {country greece, yearOfBirth 1980, married TRUE, nOfChildren 3}


	    EquivGreekCitizenTwoChildren ::= Citizen (WITH COMPONENTS {...,
					country(greece),
					nOfChildren (2)})


	    bGreekCitizenWithTwoChildren EquivGreekCitizenTwoChildren ::= {country greece, yearOfBirth 1980, married TRUE, nOfChildren 2}


END

