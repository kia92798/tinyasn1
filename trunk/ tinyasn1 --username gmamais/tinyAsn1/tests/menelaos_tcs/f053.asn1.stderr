Warning: value 'aGreekCitizenWithTwoChildren3'defined in line 19 does not conform to its type constraints
